_docs-config/mkdocs.yml

```diff
@@ -0,0 +1,74 @@
+site_name: ostis-ps-lib
+dev_addr: 127.0.0.1:8021
+repo_url: https://github.com/ostis-ai/ostis-ps-lib
+edit_uri: edit/main/docs/
+
+site_dir: ../../site
+docs_dir: ..
+
+exclude_docs: |
+  /README.md
+  build
```

Comment:
can sources and build folders of components be excluded?

Reply:
Build folders of components are excluded yet.